<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formProfileManager</class>
 <widget class="QMainWindow" name="formProfileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>375</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Proxies Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <activeon>:/newPrefix/rabbit.ico</activeon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>:/icon_24x24/icon_24x24/add_24x24.png</selectedon>
     </iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>:/icon_512x512/icon_512x512/edit.png</selectedon>
     </iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>:/icon_24x24/icon_24x24/trash-circle-red-5121.png</selectedon>
     </iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteAll">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete All</string>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>:/icon_24x24/icon_24x24/trash_24x24.png</selectedon>
     </iconset>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbProxies">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>601</width>
      <height>301</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Proxy Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Values</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../resource/source.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tbProxies</sender>
   <signal>itemDoubleClicked(QTableWidgetItem*)</signal>
   <receiver>btnEdit</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
